benchmarks/dynamo/torchbench.py

```diff
@@ -382,6 +382,16 @@ def load_model(
         if self.args.trace_on_xla:
             # work around for: https://github.com/pytorch/xla/issues/4174
             import torch_xla  # noqa: F401
+
+        # Turning off kv cache for torchbench models. This is not the right
+        # thing to do, but the torchbench models are way outdated, and since we
+        # are using torchbench pt2 dashboard to track regressions (rather than
+        # improving performance), we are just setting the kv cache to false.
+        # Real transformers benchmarks will be added soon using a different
+        # infra.
+        if model_name.startswith("hf") and hasattr(model.config, "use_cache"):
+            model.config.use_cache = False
```

Comment:
what would be the issue if we don't turn off kv cache? We probably want to track a real-world setting for regression..